yarn.lock

```diff
@@ -5120,7 +5120,14 @@ rxjs@6.5.4:
   dependencies:
     tslib "^1.9.0"
 
-rxjs@^6.5.3, rxjs@^6.5.5:
+rxjs@7.0.0-beta.0:
+  version "7.0.0-beta.0"
+  resolved "https://registry.yarnpkg.com/rxjs/-/rxjs-7.0.0-beta.0.tgz#41f730761e00ba7b5a28955e0f177afba8e2a134"
+  integrity sha512-MMsqDczs2RzsTvBiH6SKjJkdAh7WaI6Q0axP/DX+1ljwFm6+18AhQ3kVT8gD7G0dHIVfh5hDFoqLaW79pkiGag==
+  dependencies:
+    tslib "^1.9.0"
+
+rxjs@^6.5.3:
```

Comment:
I think this means one of our dependencies still lead to rxjs@6.6.0. We may want to identify and make sure it is fine.

Reply:
The rxjs@6.5.3, 6.5.4 versions in the yarn.lock are only used from `@angular/cli`.  Although I confirmed that yarn install still installs rxjs 7 beta and the app builds/runs fine, it's possible some breaking change in rxjs 7 beta may silently affect `@angular/cli`.